src/Native/System.Security.Cryptography.Native/configure.cmake

```diff
@@ -7,8 +7,6 @@ set(CMAKE_REQUIRED_INCLUDES ${OPENSSL_INCLUDE_DIR})
 check_library_exists(${OPENSSL_SSL_LIBRARY} "TLSv1_1_method" "" HAVE_TLS_V1_1)
```

Comment:
When I read it I was thinking it was actually seeing how far back it could reach, instead of how far forward.

So while this probably could be deleted, it might be valueable in case TLS 1.0 support gets removed by anywhere we run, and we need to keep around the reminder of how this stuff works :).